src/EnergyPlus/HVACSingleDuctInduc.cc

```diff
@@ -250,10 +251,10 @@ namespace HVACSingleDuctInduc {
         // the tasks usually done by the Update and Report routines are not required in a compound terminal unit.
 
         // Update the current unit's outlet nodes. No update needed
-        // CALL UpdateIndUnit(IUNum)
+        // CALL UpdateIndUnit(IUNum);
 
         // Fill the report variables. There are no report variables
-        // CALL ReportIndUnit(IUNum)
+        IndUnit(IUNum).ReportIndUnit();
     }
```

Comment:
The new member function added for FourPipeInduction air terminal unit is called from Report Function. Report function was added for this purpose. The new function needs to be calculated only once every simulation of the air terminal unit.